src/Commands/ModuleMakeCommand.php

```diff
@@ -68,8 +69,14 @@ protected function getOptions()
     {
         return [
             ['plain', 'p', InputOption::VALUE_NONE, 'Generate a plain module (without some resources).'],
+            ['api', null, InputOption::VALUE_NONE, 'Generate an api module.'],
+            ['web', null, InputOption::VALUE_NONE, 'Generate a web module.'],
             ['disabled', 'd', InputOption::VALUE_NONE, 'Do not enable the module at creation.'],
             ['force', null, InputOption::VALUE_NONE, 'Force the operation to run when the module already exists.'],
         ];
     }
+
+    private function getModuleType($isPlain,$isApi,$isWeb){
+        return ($isPlain && $isApi && $isWeb)||($isPlain && $isApi)||($isApi && $isWeb)||($isPlain && $isWeb)?'web':($isPlain&&(!$isApi&&!$isWeb)?'plain':($isApi&&(!$isPlain&&!$isWeb)?'api':'web'));
```

Comment:
I have no clue wtf this does.

Reply:
it just returns web if users entered more that one options , --plain --web --api

Reply:
yes I guessed that, but it's unreadable. The new version is still unreadable. Don't use ternaries here.
Also please create tests for each possible outcome.